test/python/test_pack.py

```diff
@@ -445,7 +445,7 @@ def test_tempdir(cartridge_cmd, project_without_dependencies, tmpdir, pack_forma
     assert re.search(r'Temporary directory is set to {}'.format(cartridge_tempdir), output) is not None
 
 
-@pytest.mark.parametrize('pack_format', ['tgz', 'deb'])
```

Comment:
Wy do you drop "deb"?

Reply:
Because this part is common for all types.

Reply:
Is it really consume a lot of time?

It's just for now and easily could be changed in future